docs/posts/run-on-snowflake/index.qmd

```diff
@@ -0,0 +1,147 @@
+---
+title: "Ibis - Now flying on Snowflake"
+author:
+  - Phillip Cloud
+  - Tyler White
+error: false
+date: "2024-06-19"
+categories:
+  - blog
+  - new feature
+  - snowflake
+---
+
+Ibis allows you to push down compute operations on your data where it lives,
+with the performance being as powerful as the backend you're connected to. But
+what happens if Ibis is running _inside_ the backend you're connected to?
+
+In this post, we will discuss how we got Ibis running on a Snowflake virtual
+warehouse.
+
+## Why would we want to do this?
+
+Snowflake has released several features to enable users to execute native
+Python code on the platform. These features include a new notebook development
+interface, Streamlit in Snowflake, the Native App framework, and Python within
+functions and stored procedures.
+
+If users could use Ibis directly within the platform, developers could more
+easily switch between a local execution engine during development and
+efficiently deploy and operationalize that same code on Snowflake.
+
+But this isn't without its challenges; there were a few things we needed to
+figure out, and these are the questions we will answer throughout the post.
+
+- How can we get an Ibis connection to Snowflake - from in Snowflake?
```

Comment:
```suggestion
- How can we get an Ibis connection to Snowflake - from within Snowflake?
```

or `inside`, maybe?

Reply:
I like the usage of "within" here. That is a bit clearer than using "from in" together.  

Thank you for the review!